cos-fleetshard-sync/src/main/java/org/bf2/cos/fleetshard/sync/resources/ConnectorStatusUpdater.java

```diff
@@ -87,9 +89,11 @@ public void update(ManagedConnector connector) {
     }
 
     /*
-     * Expose a metric "cos_fleetshard_sync_connector_state" which reveals the current connector state.
+     * Expose a Gauge metric "cos_fleetshard_sync_connector_state" which reveals the current connector state.
      * Metric value of 1 implies that the connector is in Ready state. Similarly, 2 -> Failed, 3 -> Deleted,
      * 4 -> Stopped, 5 -> In Process
+     * Also exposing a Counter metrics "cos_fleetshard_sync_connector_state_count_total" which reveals each
```

Comment:
where is this exposed ? is it automatically exposed from the gauge ?

Reply:
micrometer appends `total` for counters I believe. `cos_fleetshard_sync_connector_state_count` in defect get exposed as `cos_fleetshard_sync_connector_state_count_total`